### PR TITLE
test ci: get t-label

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: scripts/get_tlabel.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+>&2 printf $'Using PR: \'%s\'\n' "${PR}"
+
+tlabels="$(gh api --jq '.labels.[].name' "${PR}" | grep -- '^t-' || printf 'generic')"
+# print to error, since the stdout is captured into `GITHUB_OUTPUT
+>&2 printf 't-labels:\n---\n%s\n---\n' "${tlabels}"
+# if there isn't exactly 1 `t-xxx` label, use `generic`
+if [[ "$(wc -l <<<"${tlabels}")" -ne 1 ]]; then
+  tlabels="generic"
+fi
+streamName="${tlabels} maintainer merges"
```
Comment (jcommelin) [inline scripts/get_tlabel.sh:14]:
```suggestion
topicName="${tlabels} maintainer merges"
```

File: .github/workflows/bors.yml [hunk 2]
```diff
@@ -26,6 +26,30 @@ jobs:
...
+          if [[ "$(wc -l <<<"${tlabels}")" -ne 1 ]]; then
+            tlabels="generic"
+          fi
+          streamName="${tlabels} maintainer merges"
+          >&2 printf $'Post to topic: \'%s\'"\n' "${streamName}"
+          echo "name=${streamName}"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR:  /repos/{owner}/{repo}/issues/16350 #{pull_number}
+
```
Comment (jcommelin) [inline .github/workflows/bors.yml:52]: Do we need all these changes to all these `yml` files?

**Comment by adomani** [comment]
maintainter merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Comment by adomani** [comment]
maintainer merge

**Pull request closed without merging**